pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>symbol-annotation</artifactId>
-      <version>1.7</version>
+      <version>1.17</version>
```

Comment:
We'd rather keep this at `1.7` as this dep also introduced by the `credentials` dep, at a lower version.

Reply:
I updated the version because it grabs `annotation-indexer` in an older version than the one grabbed by `jenkins-core` in `2.60.3`. We can resolve that with a `dependencyManagement` but I think it's overkill.

There is no problem using a more recent version of `symbol-annotation` than the one grabbed by `credentials`. Or else, no one will ever upgrade versions in  plugin development ;)

Reply:
Makes sense, thanks!